src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseHelpAndSupportSection.tsx

```diff
@@ -74,6 +75,11 @@ const ReleaseHelpAndSupportSection = ({
             <NationalStatisticsSection />
           </AccordionSection>
         )}
+        {release.type && release.type.title === ReleaseType.OfficialStatistics && (
```

Comment:
Is `type` optional? Could make this more succinct with `release.type.title === ReleaseType.OfficialStatistics`. Think the same applies for the section above too.

If it is supposed to be optional, then I would suggest changing the corresponding field on `Release` optional, and then express this as `release.type?.title === ReleaseType.OfficialStatistics` (using optional chaining)